doc/man3/flux_fd_watcher_create.rst

```diff
@@ -36,7 +36,7 @@ to monitor for events on a file descriptor *fd*. When events occur,
 the user-supplied *callback* is invoked.
 
 The *events* and *revents* arguments are a bitmask containing a logical
-“or” of the following bits. If a bit is set in *events*, it indicates
+OR the following bits. If a bit is set in *events*, it indicates
```

Comment:
The wording no longer makes sense as it now says "a logical OR the following bits". Maybe the `of` was inadvertently dropped?

Reply:
doh! thanks.